test/vector-tests/resouces/VotingKeyTestVector.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
I'll move this file to https://github.com/symbol/test-vectors

maybe 7.test-voting.json.

use camelCase for properties...

Reply:
to generate these keys, a set of deterministic key pairs is required as seen in the test,  not sure how easy this can be converted to static keys in the vector.

Reply:
It's just moving the files to the vectors repo, then this sdk and other sdks will test the same input validating they all work the same